src/abstractdataframe/abstractdataframe.jl

```diff
@@ -811,10 +815,14 @@ julia> dropmissing(df, [:x, :y])
 """
 function dropmissing(df::AbstractDataFrame,
                      cols::Union{ColumnIndex, MultiColumnIndex}=:;
-                     disallowmissing::Bool=true)
-    newdf = df[completecases(df, cols), :]
+                     view::Bool=false, disallowmissing::Bool=!view)
+    view && disallowmissing && throw(ArgumentError("it is not possible to disallow " *
+                                                   "missing when view=true"))
```

Comment:
Maybe more to the point:
```suggestion
    view && disallowmissing &&
        throw(ArgumentError("disallowmissing=true is incompatible with view=true"))
```